docs/Worker-Managers.md

```diff
@@ -0,0 +1,59 @@
+# Worker Manager Setup
+
```

Comment:
Add a paragraph here explaining what a worker manager is, as well as a table listing the types of supported worker managers (with columns "name" and "description")